collect_app/src/androidTest/java/org/odk/collect/android/feature/external/AndroidShortcutsTest.kt

```diff
@@ -56,4 +58,10 @@ class AndroidShortcutsTest {
         assertThat(shortcutTargetIntent.action, equalTo(Intent.ACTION_EDIT))
         assertThat(shortcutTargetIntent.data, equalTo(FormsContract.getUri("DEMO", formId)))
     }
+
+    @Test
+    fun blah() {
```

Comment:
What is this? :smile:

Reply:
haha doh! That's me checking if something works as I expect and then forgetting to get rid of it.